File: trunk/src/baseball.lib/baseball.lib/model/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using com.mxply.net.common.Extensions;

namespace com.mxply.app.baseball.lib.model
{
    public partial class Person
    {
        public override string ToString()
        {
            return string.Format("{0}, {1}", this.LastName.ToUpper(), this.FirstName.ToCorrectCase());
        }
    }
}

File: trunk/src/baseball.client.droid/src/com/mxply/app/baseball/client/droid/model/Person.java
package com.mxply.app.baseball.client.droid.model;

import java.util.UUID;

public class Person 
{
	public static Class<Person> PERSON_CLASS = Person.class;
	
	UUID id;
	String firstName;
	String lastName;
	
	public UUID getId() {
		return id;
	}
	public void setId(UUID id) {
		this.id = id;
	}
	public String getFirstName() {
		return firstName;
	}
	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}
	public String getLastName() {
		return lastName;
	}
	public void setLastName(String lastName) {
		this.lastName = lastName;
	}
	
	@Override
	public String toString() {
		return "Person [firstName=" + firstName + ", lastName=" + lastName
				+ "]";
	}
}
